packages/react-components/src/components/MessageThread.tsx

```diff
@@ -625,6 +625,12 @@ export type MessageThreadProps = {
    * @beta
    */
   onDisplayDateTimeString?: (messageDate: Date) => string;
+  /* @conditional-compile-remove(file-sharing) */
+  /**
+   * Optional arialabel strings for file download cards
+   * @beta
+   */
+  fileDownloadStrings?: _FileDownloadCardsStrings;
```

Comment:
Let's not introduce a new prop.
The prop `strings` already exists for holding MessageThread localization strings

Reply:
done